Helper/Api.php

```diff
@@ -174,7 +180,7 @@ private function getFullApiUrl($dynamicUrl)
      */
     private function isResponseError($response)
     {
-        $arr = (array)$response;
+        $arr = (array) $response;
```

Comment:
What is the class or type of `$response` variable?

If `$reponse` extend from `Magento\Framework\DataObject` so we can call
`$response->getErrors()` (or `$response->hasErrors()` depends from object) and `$response->getErrorCode()`.

I think it will be more correct instead of convert object to array.

Reply:
I don't want to change the functionality here in this PR, this is again my linter just making changes